rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -205,13 +253,19 @@ def get_balances(
                             'is_lp': False,
                         })
                     # Always write detected nfts in the DB to have name and address associated
-                    db_data.append((nft.token_identifier, nft.name, None, None, 0, address))  # noqa: E501
+                    db_data.append((nft.token_identifier, nft.name, None, None, False, address, False, nft.image_url, str(ZERO), collection_name))  # noqa: E501
         # save opensea data in the DB
+        fresh_nfts_identifiers = [x[0] for x in db_data]
         if len(db_data) != 0:
             with self.db.user_write() as cursor:
+                cursor.execute(
+                    f'DELETE FROM nfts WHERE identifier NOT IN '
+                    f'({",".join("?"*len(fresh_nfts_identifiers))})',
+                    fresh_nfts_identifiers,
```

Comment:
Alexey can you add a comment for why you are doing this?

For the reader the nft table is not referenced by any other table so this wont raise any error deleting. By doing this we are invalidating the local cache if the user transfers any NFT